### PR TITLE
Create locales for Gahuza and Tigrinya

### LINKED ISSUE #2180: Create moment locale for - Gahuza, Tigrinya
**Is your feature request related to a problem? Please describe.**
Moment doesn't have a locale for Hausa and Somali.

**Describe the solution you'd like**
Create a locale for Gahuza (rw) and Tigrinya (yi) in `psammead-locales`. 
Get the locales [here](https://repl.it/repls/ComplicatedCluelessTab)

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #2180

**Overall change:**
 Add moment locales for Gahuza (rw) and Tigrinya (ti).

**Code changes:**
- Add locale files with months and relative time
- Add unit tests.

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] Automated jest tests added (for new features) or updated (for existing features)
- [X] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-locales/moment/index.stories.jsx [hunk 1]
```diff
@@ -46,10 +48,11 @@ const locales = [
   { name: 'Serbian', locale: 'sr' },
   { name: 'Serbian Cyrillic', locale: 'sr-cyrl' },
   { name: 'Tamil', locale: 'ta' },
+  { name: 'Gahuza', locale: 'ga' },
```
Comment (ghost) [inline packages/utilities/psammead-locales/moment/index.stories.jsx]:
```suggestion
  { name: 'Gahuza', locale: 'rw' },
```

**Review by ghost — CHANGES_REQUESTED** [review]
Include the correct locale for `Gahuza` in storybook

File: packages/utilities/psammead-locales/moment/index.stories.jsx (hunk 1 above)
Comment (DenisHdz) [reply packages/utilities/psammead-locales/moment/index.stories.jsx]: You have been faster than me replacing it :P

**Comment by ghost** [comment]
Could you check why Jenkins is failing.

**Comment by AlistairGempf** [comment]
Is there a reason for the Tigrinya locale being named `yi` rather than `ti` as it is in the spreadsheet and in the language data?

**Comment by DenisHdz** [comment]
> Is there a reason for the Tigrinya locale being named `yi` rather than `ti` as it is in the spreadsheet and in the language data?

You are right, I have created it based on what the issue says. In the [moment-ws](https://github.com/bbc/moment-ws/tree/master/src/locale) repo we have `ti`.
@sadick254 Was it a typo? 😁

**Comment by ghost** [comment]
> Was it a typo?

It was 😱 apologies for that @DenisHdz

File: packages/utilities/psammead-locales/moment/ti.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+var moment = require('moment');
+
+moment.defineLocale('yi', {
```
Comment (AlistairGempf) [inline packages/utilities/psammead-locales/moment/ti.js]:
```suggestion
moment.defineLocale('ti', {
```

Comment (DenisHdz) [reply packages/utilities/psammead-locales/moment/ti.js]: Hopefully, this will be the last `yi` to replace :P

**Review by AlistairGempf — APPROVED** [review]
I did a cmd-f for `yi` and found nothing so I think we're all good ;)

**Review by ghost — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**